libs/domains/organizations/feature/src/lib/hooks/use-create-container-registry/use-create-container-registry.tsx

```diff
@@ -0,0 +1,27 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query'
+import { mutations } from '@qovery/domains/organizations/data-access'
+import { queries } from '@qovery/state/util-queries'
+
+export interface UseCreateContainerRegistryProps {
+  organizationId: string
+}
+
+export function useCreateContainerRegistry({ organizationId }: UseCreateContainerRegistryProps) {
+  const queryClient = useQueryClient()
+
+  return useMutation(mutations.createContainerRegistry, {
+    onSuccess() {
+      queryClient.invalidateQueries({
+        queryKey: queries.organizations.containerRegistries({ organizationId }).queryKey,
+      })
+    },
```

Comment:
Should we invalidate `availableContainerRegistry` too ?

Reply:
No, because it's only available to build the form with the types, to create a container registry and it should not change with each update or creation